src/core/FilePath.cpp

```diff
@@ -211,6 +211,9 @@ FilePath::FilePath()
     else if (testSetDir(appDirPath + "/share")) {
     }
 #endif
+    // Last ditch test when running in the build directory (mainly for travis tests)
+    else if (testSetDir(QString(KEEPASSX_SOURCE_DIR) + "/share")) {
```

Comment:
Wow didnt even remember that i flopped it from debug to release! In either case both builds are now tested in travis which is good because they are built differently.